client/signup/jetpack-connect/index.jsx

```diff
@@ -180,7 +184,7 @@ const JetpackConnectMain = React.createClass( {
 					<p>{ this.translate( 'We need to send you to your WordPress install so you can approve the Jetpack installation. Click the button in the bottom-right corner on the next screen to continue.' ) }</p>
```

Comment:
We have an issue around for someone from growth or marketing to review all the texts in here ... I guess most of them are going to be changed